friends/package.json

```diff
@@ -4,6 +4,8 @@
   "private": true,
   "dependencies": {
     "axios": "^0.18.0",
+    "customHooks": "https://github.com/brellin/customHooks.git",
```

Comment:
Just add a custom Hook file in this project and import it to the component that need it